src/TSTransformer/macros/callMacros.ts

```diff
@@ -60,5 +62,10 @@ export const CALL_MACROS: MacroList<CallMacro> = {
 		return luau.binary(luau.binary(successId, "and", successExp), "or", failureExp);
 	},
```

Comment:
Since 4.6, this macro is practically useless, now that the relation between the pcall returns is kept. Because renaming is a breaking change, the change might as well be to remove it altogether.

Reply:
If it is kept in, that last return will look nicer with the new Luau if-then-else expression